mesonbuild/modules/python.py

```diff
@@ -297,6 +300,55 @@ def __init__(self, interpreter: 'Interpreter') -> None:
             'find_installation': self.find_installation,
         })
 
+    def _get_install_scripts(self) -> T.List[mesonlib.ExecutableSerialisation]:
+        backend = self.interpreter.backend
+        ret = []
+        optlevel = self.interpreter.environment.coredata.get_option(mesonlib.OptionKey('bytecompile', module='python'))
+        if optlevel == -1:
+            return ret
+
+        installdata = backend.create_install_data()
+        py_files = []
+
+        def should_append(f, isdir: bool = False):
+            return f.startswith(('{py_platlib}', '{py_purelib}')) and (f.endswith('.py') or isdir)
```

Comment:
It took me a while to understand this. Would be nice to add a comment telling that out_name and install_path_name are from P_OBJ.OptionString() returned by _get_install_dir_impl(). Not sure how to formulate it, but some hint would help understanding this code in the future.

Reply:
Okay, done.